auth/facebooklogin.js

```diff
@@ -28,15 +28,16 @@ module.exports = function(app,passport) {
         clientID        : config.SECRETS.facebook_auth.client_ID,
         clientSecret    : config.SECRETS.facebook_auth.client_secret,
         callbackURL     : config.SECRETS.facebook_auth.callbackURL,
-        profileFields   : ['id', 'displayName', 'name', 'gender' , 'email', 'photos']
+        profileFields   : ['id', 'displayName', 'name', 'gender' , 'email', 'photos'],
+        enableProof: true
     },
```

Comment:
What is the use of this?

Reply:
it is used for making app secret proof to make facebook app more secure

Reply:
but as we are  not using graph api there is no use

Reply:
https://www.sammyk.me/the-single-most-important-way-to-make-your-facebook-app-more-secure

Reply:
Good stuff

Reply:
so should i remove the line or will it be merged?

Reply:
I'll merge it tonight

Reply:
squash toh nhi karna i guess?

Reply:
No need to squash commits here